README.md

```diff
@@ -229,7 +229,7 @@ Download the binary from the [releases page](https://github.com/aviator-co/av/re
 | `av branch`        | Create a new child branch from the current branch.   |
 | `av commit amend`  | Amend the last commit and rebase the children.       |
 | `av commit split`  | Split the last commit.                               |
-| `av pr create`     | Create or update a PR.                               |
+| `av pr`     | Create or update a PR.                               |
```

Comment:
This is fixed in the next PR, preventing myself a headache by trying to fix it in this one now.